apps/web/src/views/PoolDetail/components/PoolTokens.tsx

```diff
@@ -0,0 +1,62 @@
+import { Flex, RowBetween, Text } from '@pancakeswap/uikit'
+import { useMemo } from 'react'
+import { PoolInfo } from 'state/farmsV4/state/type'
+import { useChainNameByQuery } from 'state/info/hooks'
+import styled from 'styled-components'
+import { formatAmount } from 'utils/formatInfoNumbers'
+import { CurrencyLogo } from 'views/Info/components/CurrencyLogo'
+
+const StyledPoolTokens = styled(Flex)`
+  border: 1px solid ${({ theme }) => theme.colors.cardBorder};
+  background-color: ${({ theme }) => theme.colors.background};
+  padding: 16px;
+  flex-direction: column;
+  gap: 8px;
+  margin-top: 8px;
+  border-radius: 16px;
+`
+
+const formatOptions = {
+  displayThreshold: 0.001,
+}
+
+type PoolTokensProps = {
+  poolInfo?: PoolInfo | null
+}
+export const PoolTokens: React.FC<PoolTokensProps> = ({ poolInfo }) => {
+  const chainName = useChainNameByQuery()
+  const [token0Tvl, token1Tvl] = useMemo(() => {
+    if (!poolInfo) return [0, 0]
+    return [
+      formatAmount(Number(poolInfo.tvlToken0 ?? 0), formatOptions),
+      formatAmount(Number(poolInfo.tvlToken1 ?? 0), formatOptions),
+    ]
+  }, [poolInfo])
```

Comment:
use poolInfo.tvlToken0 & poolInfo.tvlToken1 as deps will be better to reduce reCalc